tracpro/templates/baseline/baselineterm_chart.html

```diff
@@ -0,0 +1,47 @@
+<script src="http://code.highcharts.com/modules/exporting.js"></script>
+<script>
+$(function () {
+    $('#container').highcharts({
+        title: {
+            text: '{{ baselineterm.name|escapejs }}'
+        },
+        subtitle: {
+            text: 'Baseline: {{ baselineterm.baseline_question|escapejs }}, Follow Up: {{ baselineterm.follow_up_question|escapejs }}'
+        },
+        xAxis: {
+            categories: [
+                        {% for date in date_list %}
+                            '{{ date }}',
+                        {% endfor %}
+                        ]
+        },
+        yAxis: {
+            title: {
+                text: '{{ baselineterm.y_axis_title|escapejs }}'
+            }
+        },
+        series: [
+        // Baseline Data
+        {% for key, values in baseline_dict.items %}
+            {
+                type: 'area',
+                name: '{{ key|escapejs }} Baseline',
+                {% for subkey, subvalue in values.items %}
+                    data: {{ subvalue }}
+                {% endfor %}
+            },
+        {% endfor %}
+        // Follow Up Data
+        {% for key, values in answers_dict.items %}
+            {
+                type: 'area',
+                name: '{{ key|escapejs }} Follow Up',
+                {% for subkey, subvalue in values.items %}
```

Comment:
Since you're only using the value, you could use `{% for subvalue in values.values %}`.